### ISSUE TITLE
Accessibility Issue:: Select box without a description

### ISSUE THREAD

**Issue by ddavisgraphics** [wvulibraries/databases_deprecated#89]
'Select box' without a description

Make a decision for this issue
Drop down menus (select boxes or role="listbox") should always have a description that is explicitly associated with the field to make sure that users of assistive technologies will also know what the field is for.

If the drop down menu (select box or role="listbox") has a visible description indicating the purpose of the menu, this description should be explicitly associated to the input drop down menu either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.

If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the drop down menu or create an invisible label using the WAI-ARIA attribute 'aria-label'.
```<!-- html -->
<ol class="mp_list" aria-atomic="true" aria-busy="false" aria-live="polite"
id="mp_q_list" role="listbox" style="display: none; width: 186.3px; padding-left:
10.35px; padding-right: 10.35px;"></ol>
```

**Comment by ddavisgraphics** [comment]
Should be fixed by https://github.com/wvulibraries/databases/commit/710c4f6d0355aa42f19b207afea03ed7ad9f7af4
